### DISCUSSION TITLE
`percy exec --` results in a cannot finalize before all snapshot resources are uploaded error

### DISCUSSION THREAD

**Discussion by it-ony** [percy/cli#772] [Support]
## The problem

Running `percy exec -- mvn --no-transfer-progress -Dbrowser=chrome '-Dit.test=!MultipleBrowsersDocumentUploadIT' -DthreadCount=10 -Denvironment=headless clean verify`

```
Error: Finalizing build 15549364 failed: cannot finalize before all snapshot resources are uploaded. This is likely a client error, please make sure that content for all SHAs in 'missing-resources' from the snapshot response are uploaded before calling finalize.
```

See the error as a github action [here](https://github.com/onfido/onfido-sdk-ui/runs/5111378794?check_suite_focus=true#step:22:5071).

## Environment

- Node version: 14.18.3
- `@percy/cli` version: @percy/cli@1.0.0-beta.74
- Version of Percy SDK you’re using: n/a
- If needed, a build or snapshot ID: https://github.com/onfido/onfido-sdk-ui/runs/5111378794
- OS version: ubuntu  20.04.3
- Type of shell command-line [interface]: bash

## Details

I'm running `percy exec --` ... as part of the pipeline since I while, and this is the 1st time, I'm seeing this error.

## Code to reproduce issue

Sorry, no easy to reproduce project here. Failed only once in the pipeline so far. Was working in one of the next builds again, e.g. https://github.com/onfido/onfido-sdk-ui/runs/5113990018?check_suite_focus=true

**Comment by Robdel12** [comment]
Hey @it-ony! Thanks for the detailed report! Could you run a build with a `--verbose` flag to get the full debug logs? That'll hint towards what's going wrong. https://docs.percy.io/docs/debugging-sdks#debug-vs-verbose-logging
